core/storage/base_model/gae_models.py

```diff
@@ -83,7 +91,7 @@
 ID_LENGTH = 12
 
 
-class BaseModel(datastore_services.Model):
+class BaseModel(datastore_services.Model): # type: ignore[misc]
```

Comment:
Are `misc` type of errors frequent?

Reply:
The errors are not that frequent. This is here due to type of datastore_services.Model being of type Any.

Reply:
Why is the type of datastore_services.Model equal to Any? That seems weird.

Using Any is an antipattern and every usage should be carefully justified. Often there will be ways to work around it. We had similar issues last year in the typescript project and managed to remove most cases.

Reply:
> Why is the type of datastore_services.Model equal to Any? 
datastore_services.Model is ndb.Model. And ndb doesn't have type  stubs or type annotations. I have tried to reduce the usage of `Any` and tried to find workarounds whenever possible.

Reply:
These attempts at workarounds should be fully documented. Please use the doc that was created for this purpose and get sign-off before resorting to Any.

Reply:
Should I add this to doc now or is it approved? 
I'll take care of this from next time.

Reply:
It's not approved. I still need to see the justification.

Reply:
Added stubs now